.vscode/launch.json

```diff
@@ -622,22 +621,31 @@
       "internalConsoleOptions": "openOnSessionStart"
     },
     {
-        "name": "Debug Faux Jest Unit Tests",
-        "cwd": "${workspaceRoot}/packages/salesforcedx-sobjects-faux-generator",
-        "type": "node",
-        "request": "launch",
-        "runtimeArgs": ["--inspect-brk", "${workspaceRoot}/packages/salesforcedx-sobjects-faux-generator/node_modules/.bin/jest", "--runInBand", "--coverage", "false"],
-        "console": "integratedTerminal",
-        "internalConsoleOptions": "neverOpen"
-    }, 
-    {
-      "name": "Debug Apex Debug Jest Unit Tests",
+      "name": "Debug Tests - Apex Debug - Unit Tests",
```

Comment:
Standardized the naming of the unit test configs to mach the "Launch Tests" configs.